packages/mui-utils/src/getScrollbarSize.ts

```diff
@@ -1,7 +1,9 @@
+import ownerWindow from './ownerWindow';
+
 // A change of the browser zoom change the scrollbar size.
 // Credit https://github.com/twbs/bootstrap/blob/488fd8afc535ca3a6ad4dc581f5e89217b6a36ac/js/src/util/scrollbar.js#L14-L18
 export default function getScrollbarSize(doc: Document): number {
   // https://developer.mozilla.org/en-US/docs/Web/API/Window/innerWidth#usage_notes
   const documentWidth = doc.documentElement.clientWidth;
-  return Math.abs(window.innerWidth - documentWidth);
+  return Math.abs(ownerWindow(doc).innerWidth - documentWidth);
```

Comment:
The function `getScrollbarSize` is used in various locations, and it's called with `ownerDocument(node)` as an argument. An example of its usage can be found [here](https://github.com/mui/material-ui/blob/master/packages/mui-base/src/unstable_useModal/ModalManager.ts#L103). I believe this proposed change will result in the following code:

```js
return Math.abs(container.ownerDocument.defaultView.ownerDocument.defaultView.innerWidth - documentWidth);
```

However, I don't think this is the desired outcome.